File: 1 - metodosBusca/pacoteBusca_JomiHubner/src/grafo/GrafoDirigido.java

package grafo;


public class GrafoDirigido extends Grafo {

	public void criaAresta(int i, int j) {
		Vertice vi = vertices.get(i);
		Vertice vj = vertices.get(j);
		new ArestaDirigida(vi,vj);
	}

    public void criaAresta(int i, int j, int custo) {
        Vertice vi = vertices.get(i);
        Vertice vj = vertices.get(j);
        new ArestaDirigidaValorada(vi,vj, custo);        
    }
}

File: 1 - metodosBusca/python/app/core/grafo/grafo_dirigido.py
from .grafo import Grafo
from .aresta_dirigida import ArestaDirigida
from .aresta_dirigida_valorada import ArestaDirigidaValorada

class GrafoDirigido(Grafo):
    def cria_aresta(self, i, j):
        vi = self.vertices.get(i)
        vj = self.vertices.get(j)
        ArestaDirigida(vi, vj)

    def cria_aresta(self, i, j, custo):
        vi = self.vertices.get(i)
        vj = self.vertices.get(j)
        ArestaDirigidaValorada(vi, vj, custo)